### PR TITLE
new features

### PR DESCRIPTION
**Pull request by LarysaDoroshenko**

### CODE REVIEW

File: src/main/java/words/controller/UserController.java [hunk 1]
```diff
@@ -26,4 +26,10 @@ String profile(Model model) {
         return "profile";
     }
 
+    @GetMapping("/statistics")
+    String getStatistics(Model model) {
+        model.addAttribute("statistics", "My progress");
```
Comment (ArzamastsevOleksandr) [inline src/main/java/words/controller/UserController.java:31]: better call it "message" or "header"

File: src/main/webapp/WEB-INF/view/vocabulary.jsp [hunk 2]
```diff
@@ -23,6 +23,11 @@
                     </td>
                 </tr>
             </c:forEach>
+            <tr>
+                <td>
+                    <label>Size: ${vocabulary.words.size()}</label>
+                </td>
+            </tr>
```
Comment (ArzamastsevOleksandr) [inline src/main/webapp/WEB-INF/view/vocabulary.jsp:30]: First display the vocabulary details, only then list the words

**Review by ArzamastsevOleksandr — APPROVED** [review]

File: src/main/webapp/WEB-INF/view/statistics.jsp [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+<html lang="en">
+<body>
+<div>
+    <div>
+        <h1>${statistics}</h1>
+    </div>
+</div>
+</body>
+</html>
```
Comment (ArzamastsevOleksandr) [inline src/main/webapp/WEB-INF/view/statistics.jsp:11]: add a reference to the home/profile/vocabulary pages

File: src/main/webapp/WEB-INF/view/statistics.jsp [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+<html lang="en">
+<body>
+<div>
+    <div>
+        <h1>${statistics}</h1>
+    </div>
+</div>
```
Comment (ArzamastsevOleksandr) [inline src/main/webapp/WEB-INF/view/statistics.jsp]: why do we need a <div> inside <div>?

we can remove 1

File: src/main/java/words/controller/UserController.java (hunk 1 above)
Comment (LarysaDoroshenko) [reply src/main/java/words/controller/UserController.java:31]: "header" can`t be used as it is a reserved word and displays not a suggested text, but browser information

**Pull request merged**